### PR TITLE
Fix jumps on editing text inside tooltips, and probably other components

### PR DESCRIPTION
**Pull request by istarkov**
## Description

image

https://discord.com/channels/955905230107738152/1137452246876033114

## Steps for reproduction

1. click button
2. expect xyz

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio-builder/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env.example` and the `builder/env-check.js` if mandatory

### CODE REVIEW

**Review by TrySound — APPROVED** [review]

File: apps/builder/app/canvas/shared/styles.ts [hunk 1]
```diff
@@ -66,15 +66,15 @@ const helperStyles = [
     box-shadow: inset 0 0 0 1px rgba(255, 255, 255, 0.7);
   }`,
   // Has no width, will collapse
-  `[${idAttribute}]:where(:not(body)[${collapsedAttribute}="w"]) {
+  `[${idAttribute}]:where(:not(body):not([data-lexical-editor])[${collapsedAttribute}="w"]) {
```
Comment (kof) [inline apps/builder/app/canvas/shared/styles.ts:69]: this selector slowly get to wtf levels, maybe time to think of a simplification

Comment (kof) [reply apps/builder/app/canvas/shared/styles.ts:69]: please add comment what this is fixign, there is no way we will understand later on

**Review by kof — APPROVED** [review]

**Comment by GoharAkram** [comment]
I dont see the tooltip trigger "The text you can edit" for some reason. testing in Edge

**Comment by istarkov** [comment]
We have removed interactive mode in the editor, you need to set isOpen on tooltip properties

File: apps/builder/app/canvas/shared/styles.ts (hunk 1 above)
Comment (istarkov) [reply apps/builder/app/canvas/shared/styles.ts:70]: Done.

File: apps/builder/app/canvas/features/webstudio-component/webstudio-component.tsx [hunk 2]
```diff
@@ -47,14 +46,15 @@ import { setDataCollapsed } from "~/canvas/collapsed";
...
+  renderComponentWithRef,
 }: {
-  Component: AnyComponent;
-  elementRef: ForwardedRef<HTMLElement>;
-  [idAttribute]: Instance["id"];
-  [componentAttribute]: Instance["component"];
+  // Component: AnyComponent;
+  // elementRef: ForwardedRef<HTMLElement>;
+  // [idAttribute]: Instance["id"];
+  // [componentAttribute]: Instance["component"];
```
Comment (TrySound) [inline apps/builder/app/canvas/features/webstudio-component/webstudio-component.tsx:54]: Remove comments?

**Review by TrySound — APPROVED** [review]

File: apps/builder/app/canvas/shared/styles.ts (hunk 1 above)
Comment (kof) [reply apps/builder/app/canvas/shared/styles.ts:70]: comment says what it does, but not why

Comment (istarkov) [reply apps/builder/app/canvas/shared/styles.ts:75]: Fixed.

**Pull request merged**